util/testleak/leaktest.go

```diff
@@ -59,8 +59,6 @@ func interestingGoroutines() (gs []string) {
 		"github.com/pingcap/goleveldb/leveldb.(*DB).compactionError",
 		"github.com/pingcap/goleveldb/leveldb.(*DB).mpoolDrain",
 		"go.etcd.io/etcd/v3/pkg/logutil.(*MergeLogger).outputLoop",
-		// import PD will introduce another MergeLogger
-		"go.etcd.io/etcd/pkg/logutil.(*MergeLogger).outputLoop",
```

Comment:
Why remove this?

Reply:
Struct definition is now local. I did not import PD anymore.

Reply:
`pd/server/schedule/placement` => `tidb/ddl/placement`. See `placement/types.go`.

Reply:
Got.